### DISCUSSION TITLE
Are there any plans to open source the backend code?

### DISCUSSION THREAD

**Discussion by lim-kim930** [dream-num/univer#3201] [Q&A | 常见问题] [answered]
![image](https://github.com/user-attachments/assets/e381035a-c961-42f1-951f-341c0c862008)

Currently, our import and export processes rely on closed-source code. Due to various reasons, we are unable to use the deployment solution you provided.
Therefore, I would like to inquire if there are any plans to open source it.

**Comment by jikkai** [comment, marked as answer]
We currently do not plan to open-source our server-side code or the code for some of our plugins. If our closed-source approach is a concern, you are welcome to implement a similar service yourself. Using Univer's services is entirely optional.

**Comment by lim-kim930** [comment]
Well, it looks like we have to make it happen ourselves.Thanks.
